spec/common/mount_tmpfs_spec.rb

```diff
@@ -0,0 +1,175 @@
+require 'spec_helper'
+
+# Test mount commands for tmpfs
+
+describe command('mkdir /mnt/tmpfs') do
```

Comment:
I would use `mkdir -p` here. If the /mnt/tmpfs already exists (from a pervious test run) the command won't produce an error